cpp/tests/io/parquet_test.cpp

```diff
@@ -605,6 +605,59 @@ TEST_F(ParquetWriterTest, Strings)
   cudf::test::expect_metadata_equal(expected_metadata, result.metadata);
 }
 
+TEST_F(ParquetWriterTest, StringsAsBinary)
+{
+  std::vector<const char*> strings{
+    "Monday", "Wȅdnȅsday", "Friday", "Monday", "Friday", "Friday", "Friday", "Funday"};
+  std::vector<const char*> single_byte_strings{
+    "Monday", "Wednesday", "Friday", "Monday", "Friday", "Friday", "Friday", "Funday"};
```

Comment:
Do you mean unicode strings and ascii strings?

Reply:
Yes, fixed